### DISCUSSION TITLE
openai connectivity stopped working for me

### DISCUSSION THREAD

**Discussion by helenadeus** [open-webui/open-webui#1509] [General]
# Bug Report
I cannot prompt the GPT models anymore; created a new API key to make sure it's a valid key and made sure I have credits on open AI but when I look through the models, it does not show GPT models anymore whereas it used to work fine. I'm on a mac.

## Description

**Bug Summary:**
I cannot prompt the GPT models anymore; created a new API key to make sure it's a valid key and made sure I have credits on open AI but when I look through the models, it does not show GPT models anymore whereas it used to work fine. I'm on a mac pro.

**Steps to Reproduce:**
1) Activate docker
2) open localhost:3000
3) Go to settings -> connections -> paste openAI key
4) Save (shows "save was succesful")
5) Start new chat 
6) Click "select a model"

Result: only ollama models appear (no GPT models)
<img width="904" alt="Screenshot 2024-04-11 at 06 07 23" src="https://github.com/open-webui/open-webui/assets/770297/c809e7e5-945f-4583-9e71-dd1d42505da5">

**Expected Behavior:**
I expect both ollama and gpt models to be selectable

**Actual Behavior:**
Only ollama models appear

## Environment

- **Operating System:** Macbook air Sonoma, M1 chip
- **Browser (if applicable):** Chrome

## Reproduction Details

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ x] I have included the browser console logs.
- [ x] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
<img width="714" alt="Screenshot 2024-04-11 at 06 11 05" src="https://github.com/open-webui/open-webui/assets/770297/575ba722-0456-48d4-98b6-e3b378c27149">

**Docker Container Logs:**
<img width="985" alt="Screenshot 2024-04-11 at 06 12 03" src="https://github.com/open-webui/open-webui/assets/770297/ab12b0f4-5d45-4d4c-a16c-840545b4791d">

**Screenshots (if applicable):**
<img width="904" alt="Screenshot 2024-04-11 at 06 07 23" src="https://github.com/open-webui/open-webui/assets/770297/11a46d6d-ba56-43b4-843b-2eaa91f5c1a6">

## Installation Method

Docker

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Unable to reproduce the issue here: 
<img width="772" alt="image" src="https://github.com/open-webui/open-webui/assets/25473318/4f124a01-aac0-401f-9d22-16edfd11247e">


Could you share a screenshot of what the connection tab in settings looks like? Thanks!
